package.json

```diff
@@ -24,20 +24,13 @@
     "lint": "eslint --fix index.js"
   },
   "dependencies": {
-    "@acolorbright/browserslist-config": "github:acolorbright/browserslist-config#v1.0",
-    "@babel/plugin-proposal-export-namespace-from": "7.2.0",
-    "@babel/plugin-proposal-object-rest-spread": "7.2.0",
-    "@babel/plugin-proposal-optional-chaining": "7.2.0",
-    "@babel/plugin-proposal-throw-expressions": "7.2.0",
-    "@babel/plugin-syntax-dynamic-import": "7.2.0",
-    "@babel/plugin-transform-modules-commonjs": "7.2.0",
-    "@babel/plugin-transform-react-constant-elements": "7.2.0",
-    "@babel/plugin-transform-react-inline-elements": "7.2.0",
-    "@babel/preset-env": "7.2.0",
-    "@babel/preset-react": "7.0.0",
+    "@acolorbright/browserslist-config": "github:acolorbright/browserslist-config#v2.0.0",
+    "@babel/helper-plugin-utils": "7.8.3",
+    "@babel/plugin-proposal-class-properties": "7.8.3",
```

Comment:
That damn class properties proposal that has been around since 2016...

Reply:
It's finally made it to Stage 3!

Reply:
> This proposal reached Stage 3 in July 2017. Since that time, there has been extensive thought and lengthy discussion about various alternatives

Reply:
> This proposal reached Stage 3 in July 2017. Since that time, there has been extensive thought and lengthy discussion about various alternatives

Reply:
Oh what lol